### ISSUE TITLE
Peer Feedback

### ISSUE THREAD

**Issue by wvf224** [NumEconCopenhagen/projects-2023-black-scholes#2]
Hmm, there is no assignment. However, there have been made some changes to the .py file.

**Comment by krabbe98** [comment]
Question 1 is lacking proper explination.

Generally it is clear there have been put alot of work into the py file, however without adequate explination it is a bit hard to follow.

There are alot of small problems, lacking semicolons and more that means that the code is unable to run.

I can see that you include a model for optimizing portifolio but i do not see the relevant for this assignment and agaIn without proper structure and documentation it is unclear to which question you seek to implement.
